compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/window/ComposeWindow.web.kt

```diff
@@ -306,10 +306,10 @@ internal class ComposeWindow(
 
     fun resize(boxSize: IntSize) {
         // FIXME: density is not integer
-        val scaledDensity = density.density.toInt()
+        val scaledDensity = density.density
```

Comment:
```suggestion
        val scale = density.density
```

or just remove this val